prediction_market_agent_tooling/markets/omen/data_models.py

```diff
@@ -73,6 +73,10 @@ class Question(BaseModel):
     answerFinalizedTimestamp: t.Optional[DatetimeUTC] = None
     currentAnswer: t.Optional[str] = None
 
+    @property
+    def questionId(self) -> HexBytes:
```

Comment:
the convention we use is snake case for properties
```suggestion
    def question_id(self) -> HexBytes:
```